terraform/README.md

```diff
@@ -0,0 +1,4 @@
+== JIMM-v3 Module ==
+
+This is the JIMM-v3 terraform module. The idea for this module is to consume external services via offer urls.
```

Comment:
Mention that the idea is also to deploy jimm but deploy external services via Juju offers.